packages/timezone/buildSrc/generateTimezoneList.ts

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
call the directory `scripts/` a la https://github.com/palantir/blueprint/tree/develop/packages/core/scripts

Reply:
also i would suggest simply writing this file in JS to avoid the compile step and ts-node dependency. you can add typing information for VS Code through JSDoc annotations (`/** @param {string} path */` )

Reply:
Good idea, I renamed it!

Re ts: `ts-node` is only a dev dependency, so the only downside it yields is a somewhat slower circle build time from installation and the compilation `ts-node` performs. At the same time, writing in typescript feels more natural given the codebase and the convenience and safety it provides.

Do you feel strongly about switching to JS? If so, why?